src/containers/Main.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import Input from '../components/Inputs';
+import callAPI from '../services/Api';
+import CallResponse from '../components/CallResponse';
+
+export default class Main extends Component {
+  state = {
+    url: '',
+    method: 'get',
+    json: '',
+    callResponse: '{}',
+    history: []
+  }
+
+  updateMethod = (method) => {
+    this.setState({ method: method });
+  }
+  updateJSON = (json) => {
+    this.setState({ json });
+  }
+  updateURL = (url) => {
+    this.setState({ url });
+  }
+
+  submit = (event) => {
+    event.preventDefault();
+
+    // state = this.state;
+    return callAPI(this.state.url, this.state.method, this.state.json)
+      .then(res => this.setState(() => {
```

Comment:
this is good so far - if you can complete this method to add the most recent call to state/history and display the history somewhere in the app, you will have the complete RESTy functionality. 
You should use the spread operator so that the history is cumulative (you don't want to overwrite the history with each call).